pkg/hintrunner/zerohint.go

```diff
@@ -0,0 +1,119 @@
+package hintrunner
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/alecthomas/participle/v2"
+	sn "github.com/NethermindEth/cairo-vm-go/pkg/parsers/starknet"
+	zero "github.com/NethermindEth/cairo-vm-go/pkg/parsers/zero"
+)
+
+var parser *participle.Parser[IdentifierExp] = participle.MustBuild[IdentifierExp]()
+
+func GetZeroHints(cairoZeroJson *zero.ZeroProgram) (map[uint64]Hinter, error) {
+	hints := make(map[uint64]Hinter)
+	for counter, rawHints := range cairoZeroJson.Hints {
+		pc, err := strconv.ParseUint(counter, 10, 64)
+		if err != nil {
+			return nil, err
+		}
+
+		if len(rawHints) != 1 {
+			return nil, fmt.Errorf("expected only 1 hint but got  %d", len(rawHints))
```

Comment:
Since you probably have the best understanding regarding the multi-hints per single PC, could you document it here in form of a comment?
e.g. when they occur and what would they look like in (cairo0?) code.

Reply:
I had a discussion with @rodrigo-pino about this when I first started coding this PR. I'll have to check the code of the compiler again to write the comment. I was thinking of adding an explanation of this file in a documentation page anyway, and it won't hurt to have a comment here as well

Reply:
I've been going through the cairo 0 compiler but there are lots of details I'm not certain about, this one is one of them. All of the examples I've worked with have only one hint per pc, but I haven't found the place in the compiler's code that assures that's always the case when using only whitelisted hints. I'm going to leave a TODO comment to continue studying this, and in the case it was an incorrect assumption, then I'll create another PR to fix it. The structure of the code shouldn't change much.

Reply:
I've been going through the cairo 0 compiler and there are lots of details I'm not certain about. Most of the examples I've worked with only have one hint per pc, but I haven't been able to find in the compiler code how to be certain it can't be more than that. I'll leave a TODO comment to continue studying this and in the case it was an incorrect assumption I'll create another PR to fix it. I think for the first hints to be implemented we can work as it is.